### ISSUE TITLE
Unable to access pogo.scraper-all-0.5.0-alpha4.jar

### ISSUE THREAD

**Issue by OMAoxide** [jabbink/PokemonGoBot#735]
**Description:**
[I have been trying to get this to work for a while but i do not know what i am doing.]

**Steps to reproduce:**

1. [Step 1]
2. [Step 2]

**Expected behavior:**
[What should happen?]

**Actual behavior:**
[What actually happens]

**Stacktrace (If it's a crash):**
[Please use pastebin if it's too long]

**Version:**
[The version of the bot you used; when self-compiled: :exclamation: the commit hash :exclamation:]

**Operating System:**
[Your operating system]

**Java version:**
[Full output of `java -version`]

**Comment by Sieberkev** [comment]
Make sure you are in the working directory of where the JAR file is located by using the command `cd c:\pogobot\` (or wherever you have saved it)

**Comment by OMAoxide** [comment]
I just created a folder and named it pokemongobot and i put config.properties and pogo.scraper-all-0.5.0-alpha4.jar in to it. Then in terminal i typed cd desktop. then cd pokemongobot. After that i put in the command java -jar pogo.scraper-all-0.5.0-alpha4.jar
I then got "Exception in thread "main" java.io.FileNotFoundException: config.properties (No such file or directory)"

**Comment by yashendra2797** [comment]
Ok. So here's what you do. Open Terminal (or command prompt if on Windows), type `cd ` (type the space as well), then *drag and drop* the folder with the .jar file and config.properties file. Then type `java -jar `, then *drag and drop* the .jar file. IDK if drag and dropping a folder or file into the Command Prompt works on Windows, but you get the idea.

TL;DR: Open Terminal if you're on macOS or Linux, or Command Prompt if on Windows, type in `cd *full path of the directory with the. jar and config.properties file`, then type in `java -jar pogo.scraper-all-0.5.0-alpha4.jar`

This is what my Terminal looks like: 

`Yashendras-MacBook-Air:~ yashendra2797$ cd /Users/yashendra2797/Documents/Scripts/Pokemon\ Go\ Bot 
Yashendras-MacBook-Air:Pokemon Go Bot yashendra2797$ java -jar pogo.scraper-all-0.5.0-alpha4.jar`

**Comment by Cypherke** [comment]
and i quote 

> I then got "Exception in thread "main" java.io.FileNotFoundException: config.properties (No such file or directory)"

now let me quote the readme

> Rename ./config.properties.template to ./config.properties

**Comment by OMAoxide** [comment]
@yashendra2797 i did all of that and i got and heres what it looks like

kevin-moores-MacBook-Pro-2:~ kaelinmoore$ cd /Users/kaelinmoore/Desktop/PokemonGoBot 
kevin-moores-MacBook-Pro-2:PokemonGoBot kaelinmoore$ java -jar pogo.scraper-all-0.5.0-alpha4.jar
Exception in thread "main" java.io.FileNotFoundException: config.properties (No such file or directory)
	at java.io.FileInputStream.open0(Native Method)
	at java.io.FileInputStream.open(FileInputStream.java:195)
	at java.io.FileInputStream.<init>(FileInputStream.java:138)
	at java.io.FileInputStream.<init>(FileInputStream.java:93)
	at ink.abb.pogo.scraper.MainKt.main(Main.kt:79)
kevin-moores-MacBook-Pro-2:PokemonGoBot kaelinmoore$

**Comment by Sieberkev** [comment]
Make sure the filename is config.properties and not config.properties.txt

**Comment by Parry1217** [comment]
do a 'ls' (or dir) of the directory where the scraper is found and post results here.

**Comment by OMAoxide** [comment]
i thought the file was already config.properties but i was wrong. Thank you all for helping me.

**Comment by OMAoxide** [comment]
how should i put in the latitude and longitude it keeps saying failed to interpret

**Comment by Rofii** [comment]
```
latitude=50.123
longitude=-60.123
```

**Comment by Parry1217** [comment]
#london
latitude=51.4771710742996
longitude=-0.0019324966935413

**Comment by OMAoxide** [comment]
thanks

**Comment by yashendra2797** [comment]
@OMAoxide Use this for Lat/Long: http://www.gps-coordinates.net
